timelapse_tools/normalization/single_channel_percentile_norm.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import dask.array as da
+
+from .. import exceptions
+
+###############################################################################
+
+
+def single_channel_percentile_norm(
+    data: da.core.Array, min_p: float = 50.0, max_p: float = 99.8, **kwargs
```

Comment:
what are kwargs for here? Looks like they are unused

Reply:
In cases where I allow the user to provide a callable function as a parameter for a later process, I like to have the function have a catch all kwargs so that all functions can be interoperable in the case someone forgets to remove a parameter from a previous run.